src/TMainConsole.cpp

```diff
@@ -120,6 +120,16 @@ std::optional<QString> TMainConsole::getLabelStyleSheet(const QString& name) con
     return {};
 }
 
+std::optional<QSize> TMainConsole::getLabelSizeHint(const QString& name) const
+{
+    QMap<QString, TLabel*>::const_iterator it = mLabelMap.constFind(name);
+    if (it != mLabelMap.cend() && it.key() == name) {
+        return it.value()->sizeHint();
+    }
+
```

Comment:
ℹ️ The Qt *Java-style* methods are marginally slower than the STL-like ones but are much simpler to type and use:
```suggestion
    auto pL = mLabelMap.value(name);
    if (pL) {
        return pL->sizeHint();
    }

```

Reply:
I think it's ok to keep as it is, this bit can be performance sensitive. We use the same pattern elsewhere too.

Reply:
Not *everywhere* - **I** prefer the Java-style things - possibly because they break less easily if the `QMap` is implicitly shared **https://doc.qt.io/qt-5/containers.html#implicit-sharing-iterator-problem**.

Reply:
> I think it's ok to keep as it is, this bit can be performance sensitive. We use the same pattern elsewhere too.

I got this pattern directly from the function above it (getLabelStyleSheet) and I honestly have no personal preference in this regard.